packages/angular/projects/clr-angular/src/forms/combobox/combobox.ts

```diff
@@ -318,8 +310,8 @@ export class ClrCombobox<T> extends WrappedFormControl<ClrComboboxContainer>
   }
 
   private updateControlValue() {
-    if (this.control && this.control.control) {
-      this.control.control.setValue(this.optionSelectionService.selectionModel.model);
+    if (this.onChangeCallback) {
```

Comment:
nit: 

I will change here is `onChangeCallback` a function before trying to call like one.

We use `any` to define the type so there is no guarantee that this won't throw `is not a function error`

Reply:
Thanks! It's only public, because the interface handler is, but it's not supposed to be overwritten by the users. It gets injected by the forms API.
Not that the suggestion is bad, but I don't expect it to lead to problems. If someone overrides with something else - he'll get no worse exception than the one I could give him.